frontend/main-app/src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from "react";
+import classes from "./Navigation.module.css";
+import { DropdownMenu, Link } from "@digdir/design-system-react";
+import { MenuHamburgerIcon } from "@navikt/aksel-icons";
+import { UserDropdown } from "../UserDropdown";
+import { useTranslation } from "react-i18next";
+
+export const Navigation = (): React.JSX.Element => {
+  const { t } = useTranslation();
+  const [windowSize, setWindowSize] = React.useState(window.innerWidth);
+  const isSmallScreen = windowSize < 1000;
+
+  useEffect(() => {
+    const handleResize = () => {
+      setWindowSize(window.innerWidth);
+    };
+    window.addEventListener("resize", handleResize);
+    return () => window.removeEventListener("resize", handleResize);
+  }, []);
+
+  if (isSmallScreen) {
+    return (
+      <nav className={classes.navigation}>
+        <UserDropdown />
+        <DropdownMenu size="small">
+          <DropdownMenu.Trigger variant="secondary" icon="true">
+            <MenuHamburgerIcon className={classes.menuIcon} />
+          </DropdownMenu.Trigger>
+          <DropdownMenu.Content>
+            <DropdownMenu.Group>
+              <DropdownMenu.Item asChild>
+                <Link href="/dashboard">{t("dashboard")}</Link>
```

Comment:
We can maybe here wrap dashboard with curly braces to avoid warning. {"/dashboard"}